js/common/modules/@arangodb/testutils/replicated-logs-helper.js

```diff
@@ -413,7 +412,7 @@ const createReplicatedLog = function (database, targetConfig, replicationFactor)
 };
 
 
-const testHelperFunctions = function (database) {
+const testHelperFunctions = function (database, databaseOptions = {}) {
```

Comment:
why is that not an option of `createTestDatabase`? Is it used anywhere else that justifies this to be part of the closure?

Reply:
If I pass it to `createTestDatabase`, I also have to add it to `setUpAll`, as this calls `createTestDatabase`, which seemed inconvenient when I wrote the code.